tools/ci-build/smithy-rs-tool-common/src/changelog.rs

```diff
@@ -130,7 +131,7 @@ enum AuthorsInner {
     Multiple(Vec<String>),
 }
 
-#[derive(Clone, Debug, Default, PartialEq, Serialize, Deserialize)]
+#[derive(Clone, Debug, Default, Eq, PartialEq, Serialize, Deserialize)]
```

Comment:
Good point. Manually implemented traits in 84b0557. Didn't end up using it, but I tried updating `Authors` to have `Authors(pub(super) HashSet<String>);` so that we could just throw `#[derive(..., Eq, PartialEq, ...)]` but turns out `changelogger`'s integ test `end_to_end_changelog` failed due to a difference in the ordering of rendered author names.  So I assume `Vec` is used to make that part deterministic during tests.